legacy/firmware/ethereum_typed_data.h

```
@@ -1083,4 +1101,127 @@ static bool layoutTypedData(DisplayInfo *display_context,
+  display_info_set_value(info, chain_id_str);
+  free(chain_id_str);
+  display_info_add_item_name(info, "verifyingContract", 0);
+  uint8_t verifying_contract_bytes[20] = {0};
+  ethereum_parse_onekey(ack->verifyingContract, verifying_contract_bytes);
+  char *verifying_contract_str =
+      decode_typed_data(verifying_contract_bytes, 20, "address");
+  display_info_set_value(info, verifying_contract_str);
+  free(verifying_contract_str);
+}
+static void prepare_safe_items(DisplayInfo *info,
+                               const EthereumGnosisSafeTxAck *ack) {
```

Clang (14.0.6)

[warning] parameter 'ack' is unused

(misc-unused-parameters)